appium/webdriver/webdriver.py

```diff
@@ -691,6 +691,32 @@ def battery_info(self):
         """
         return self.execute_script('mobile: batteryInfo')
 
+    @property
+    def session_capabilities(self):
+        """ Retrieves session information from the current session
+        Usage:
+            session_capabilities = driver.session_capabilities
+        Returns:
+            `dict containing information from the current session`
+        """
+        return self.execute(Command.GET_SESSION)['value']
+
+    @property
+    def events(self):
+        """ Retrieves events information from the current session
+        Usage:
+            events = driver.events
+
+        Returns:
+            `dict containing events timing information from the current session`
+        """
+        try:
+            session_capabilities = self.session_capabilities
+            return session_capabilities['events']
+        except Exception:
+            logger.warning('Could not find events information in the session. Session: {}'.format(session_capabilities))
```

Comment:
is it expected to mute exceptions? How we can then distinguish "no events" from "cannot get any events"?

Reply:
I'll log the exception so we have more visibility on the error.